### PR TITLE
2.1.3

### PR DESCRIPTION
**Pull request by br-johndavis**
Adds code to prevent the keyboardResponding view from being offset if the viewController is being displayed in a popover.

### CODE REVIEW

File: Sources/KeyboardSupport/KeyboardScrollable.swift [hunk 1]
```diff
@@ -22,7 +22,7 @@ public struct KeyboardInfo {
     public let finalFrame: CGRect
     public let animationDuration: TimeInterval
     public let animationCurve: UInt
-    
+
```
Comment (tylermilner) [inline Sources/KeyboardSupport/KeyboardScrollable.swift]: Just to keep things tidy, could you revert all of these whitespace changes?

File: Sources/KeyboardSupport/KeyboardScrollable.swift [hunk 2]
```diff
@@ -58,35 +58,37 @@ public struct KeyboardInfo {
...
     // MARK: KeyboardScrollable Conformance
     var minimumPaddingAroundInput: UIEdgeInsets {
         return .zero
     }
-    
+
     var preservesContentInsetWhenKeyboardVisible: Bool { return true }
-    
+
+    private var isNotPopover: Bool { return popoverPresentationController == nil }
```
Comment (tylermilner) [inline Sources/KeyboardSupport/KeyboardScrollable.swift]: Do you think we could remove the negative from the variable name? I think it reads a little cleaner without it.
```swift
private var isPopover: Bool { return popoverPresentationController != nil }
```

**Review by tylermilner — CHANGES_REQUESTED** [review]

File: Sources/KeyboardSupport/KeyboardScrollable.swift (hunk 1 above)
Comment (br-johndavis) [reply Sources/KeyboardSupport/KeyboardScrollable.swift]: Gotcha, I thought removing the whitespace would be more tidy, not less!

Comment (br-johndavis) [reply Sources/KeyboardSupport/KeyboardScrollable.swift]: https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/61/commits/e3ba7670babb46efdc74bbe985614efd1768432b

File: Sources/KeyboardSupport/KeyboardScrollable.swift (hunk 2 above)
Comment (br-johndavis) [reply Sources/KeyboardSupport/KeyboardScrollable.swift]: https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/61/commits/e3ba7670babb46efdc74bbe985614efd1768432b

**Review by tylermilner — DISMISSED** [review]

**Comment by tylermilner** [comment]
This looks good to merge after we take care of a couple of housekeeping items:
- Need to add an entry in `CHANGELOG.md`
- Need to get these authors to sign the CLA on this page
image

@br-johndavis looks like you might have another GitHub account that you made commits with? @BR-Nathan-Chiu it looks like some of your commits got brought in here. Could you sign the CLA on this PR as well?

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,17 @@
 
 * None
 
+
+## 2.1.3 (2021-01-21)
+
+##### Enhancements
+
+* Adds support to not offset a KeyboardScrollable view if the viewController is presented in a popover.
```
Comment (tylermilner) [inline CHANGELOG.md:5]: Can you move this under the Master/Enhancements section and remove the `2.1.3 (2021-01-21)` header? One of us will create the versioned header when we do the release.

**Review by tylermilner — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (br-johndavis) [reply CHANGELOG.md:5]: https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/61/commits/864739ee53306fae9894e939272c87c531bb54e8

**Review by tylermilner — APPROVED** [review]

**Pull request merged**